builder/builder.go

```diff
@@ -2244,3 +2244,53 @@ Updated upstream: %d (format: %s)
 
 	return nil
 }
+
+// CheckBumpNeeded is a stub that will soon be replaced with the real version
+func (b *Builder) CheckBumpNeeded() (bool, error) {
+	return true, nil
+}
+
+// TODO: Remove once we've merged the commit this is copy/pasted from.
+func getLastBuildVersion(b *Builder) (string, error) {
+	var lastVer []byte
+	var err error
+	filename := filepath.Join(b.Config.Builder.ServerStateDir, "image/LAST_VER")
+	if lastVer, err = ioutil.ReadFile(filename); err != nil {
+		// Likely the first build or some weirdness that requires us to get the container image and build it anyway
+		return "", errors.Wrap(err, "Cannot find last built version, must get container image")
+	}
+	data := string(lastVer)
+	ver := strings.Split(data, "\n")
+
+	return ver[0], nil
+}
+
+// StageMixForBump prepares the mix for the two format bumps to be executed. The
+// current upstreamversion is saved in a backup ".bump" file, and replaced with
+// the latest version in the format range of the most recent build.
+func (b *Builder) StageMixForBump() error {
+	lastBuildVer, err := getLastBuildVersion(b)
+	if err != nil {
+		return err
+	}
+	format, first, latest, err := b.getUpstreamFormatRange(lastBuildVer)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("The upstream version for this build (%s) is outside the format range of your last mix "+
+		"(format %s, upstream versions %d to %d). This build cannot be done until you complete a "+
+		"format-bump build. Please run the following two commands to complete the format bump:\nmixer "+
+		"build format-old\nmixer build format-new\nOnce these have completed, you can re-run this build.\n",
```

Comment:
I will update my code so this is accurate, currently the correct command is build format-old --format <currFmt + 1>

Reply:
Although I kind of like it being explicit as well..